### PR TITLE
Describe AS923 region specific configuration

### PR DESCRIPTION
**Pull request by bastbu**
# PR for issue #1069

## What is being addressed

With this PR we describe region-specific configuration requirements, specifically for AS923.

### CODE REVIEW

**Review by danigian — APPROVED** [review]
Do we also have a plan to modify the Provisioning Tool accordingly? (Adding some AS923 configuration and allow the specification of these dwell limit parameters)

File: docs/user-guide/as923.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+    ...,
+    "properties": {
+        "desired": {
+            "routerConfig": { ... },
+                "desiredTxParams": {
+                "downlinkDwellLimit": true,
+                "uplinkDwellLimit": true,
+                "eirp": 5
+            }
```
[the remark this answers is not in the document]
Comment (bastbu) [reply docs/user-guide/as923.md:15]: Done in 543f71a7286a36051cffbc63f8e9005d099e5695

[the remark this answers is not in the document]
Comment (bastbu) [reply docs/user-guide/as923.md:15]: Attempt number two: 88edb0bdc04058bf0e7118edb220249c5b032188

File: docs/user-guide/as923.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+!!! warning
+    If a device is associated with multiple concentrators, it's important that all of these concentrators have the same `desiredTxParams` properties. If they do not have the same `desiredTxParams` this will lead to unpredictable and unexpected behavior. Consider using [Automatic IoT device and module management](https://docs.microsoft.com/en-gb/azure/iot-hub/iot-hub-automatic-device-management) to manage consistent configuration between multiple concentrators.
+
+The Network Server will apply the following effective dwell time settings in this order:
+
+1. If a device acknowledged a `TxParamSetupReq`, the Network Server will always apply dwell time settings based on the last reported properties of the device. If the last acknowledged dwell time settings (reported properties) do not match what is configured on the LoRa device, make sure to update the reported properties of the device to match the actual TX params.
+2. If a device has not yet acknowledged (or received) a `TxParamSetupReq`, the Network Server will apply the regional dwell time defaults for downlink transmissions until the device acknowledges a received `TxParamSetupReq` MAC command.
```
Comment (MaggieSalak) [inline docs/user-guide/as923.md:29]: What do the "the regional dwell time defaults" mean exactly - no dwell time limit?

**Review by MaggieSalak — APPROVED** [review]

File: docs/user-guide/as923.md (hunk 2 above)
Comment (bastbu) [reply docs/user-guide/as923.md]: Clarified this in ce8d735456d567024d50c8bc97b520f6ae529855. Per default dwell time limits apply if nothing else is configured in the reported properties of the device.

**Pull request merged**